internal/hcl/parser.go

```diff
@@ -568,7 +583,7 @@ func (p *Parser) EnvName() string {
 		return p.moduleSuffix
 	}
 
-	return p.ProjectName()
+	return p.envMatcher.EnvName(p.ProjectName())
```

Comment:
Not sure if there's a better way of doing this, but this seems to give me the results I want since it matches against the env names in the config file.